src/helpers/distributors/worker.js

```diff
@@ -32,8 +37,15 @@ function getTasks(config) {
  * @param {Object} config
  */
 function run(config) {
+
+  logger.updateContext({
+    runId: process.env.THUB_RUN_ID,
```

Comment:
let's switch away from setting up runId as `THUB_RUN_ID` and instead pass it through `config.runId`